### PR TITLE
Fix GitHub CI/CD bot.

### PR DESCRIPTION
**Pull request by HanFa**
**What this PR does / why we need it**:
This PR fixes the automation bot. It has been tested with another repo. We can merge this and try it out.

**Which issue(s) this PR fixes**:

Fixes #

**Describe testing done for PR**:

**Special notes for your reviewer**:

**Release note**:

```release-note

```
**New PR Checklist**

- [x] Ensure PR contains only public links or terms
- [x] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [x] Squash the commits in this branch before merge to preserve our git history
- [x] If this PR is just an idea or POC, use a [Draft PR](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests) instead of a full PR
- [x] Add appropriate [labels](https://github.com/vmware-samples/load-balancer-operator-for-kubernetes/labels) according to what type of issue is being addressed.

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+  - name: size/S
+    color: 77bb00
+  - name: size/XL
+    color: ee5500
+  - name: size/XS
+    color: "009900"
+  - name: size/XXL
+    color: ee0000
+  - name: stale
+    color: "795548"
```
Comment (XudongLiuHarold) [inline .github/labels.yml:315]: need a new line

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,315 @@
+# define labels including kinds / areas etc
+kind:
```
Comment (XudongLiuHarold) [inline .github/labels.yml:2]: Can we delete some labels and areas not suitable for us?

Like:

- mesos-flake
- friction

in kind and lots of others in area.

**Review by XudongLiuHarold — COMMENTED** [review]
Need to clean up something and make it more suitable for our repo, otherwise looks good.

**Comment by HanFa** [comment]
> Need to clean up something and make it more suitable for our repo, otherwise looks good.

Sure, I've added a new line and deleted some of the irrelevant factors, in the latest update.

**Pull request merged**